sdk/sqlmanagement/Azure.ResourceManager.Sql/api/Azure.ResourceManager.Sql.netstandard2.0.cs

```diff
@@ -10119,6 +10958,24 @@ public enum RestorePointType
         Discrete = 1,
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct RoleChangeType : System.IEquatable<Azure.ResourceManager.Sql.Models.RoleChangeType>
```

Comment:
Too generic

Reply:
Renamed to DistributedAvailabilityGroupRoleChangeType